code/modules/power/lighting.dm

```diff
@@ -303,10 +303,7 @@
 	//Light replacer code
 	if(istype(W, /obj/item/lightreplacer))
 		var/obj/item/lightreplacer/LR = W
-		if(isliving(user))

```

Comment:
This is the strangest safety check I've seen in a while--someone was apparently very afraid of ghosts somehow replacing lights.